src/d3/hapticplot.d3.spec.ts

```diff
@@ -53,3 +87,44 @@ function getPosition(element: HTMLElement, shape: string): Array<{x: number, y:
   }
   return positionArray;
 }
+
+// returns array of each generated objects color
+function getColor(element: HTMLElement, shape: string): Array<string>{
+  const childrenArray = element.querySelectorAll(shape);
+  const attrArray: Array<string> = [];
```

Comment:
The TS styleguide suggests string[] instead of Array<string> for non union types